### DISCUSSION TITLE
數位分身模擬開發

### DISCUSSION THREAD

**Discussion by Raywei0214** [1101-GS3073-C/Python-Community-GS3073-C#591] [General]
我還以為是什麼影分身之術，結果是算數據的
https://ai.cisanet.org.tw/casecontent.php?id=348

**Comment by RogelioKG** [comment]
應該是類似 Digital Twin 的東西吧，
例如在現實世界中製造一具機器人勞心勞力又容易損壞，
不如在虛擬世界中模擬出一個高度真實的物理環境，
然後我們就可以在裡面模擬出上百具機器人，調整參數去做測試，
這樣就可以省下許多時間與金錢成本。
這技術蠻常被用在工業 4.0，例如還原出一個數位工廠之類的。

**Reply by Raywei0214** [reply]
你也是位高強的點數魔人呢

**Reply by RogelioKG** [reply]
呵呵

**Comment by manipulateed** [comment]
人可能沒法每次都這麼準確地去操作，透過訓練模型能自動校正參數，減少了每次手動調整的時間，將提高整體的工作效率。
